python/cuml/test/test_nearest_neighbors.py

```diff
@@ -37,7 +37,7 @@ def stress_param(*args, **kwargs):
 
 
 @pytest.mark.parametrize('should_downcast', [True])
-@pytest.mark.parametrize('input_type', ['dataframe', 'ndarray'])
```

Comment:
Why are dataframe inputs being removed?

Reply:
@dantegd had mentioned this on his PR  : 

> Since the input_utils are now tested running this twice is not testing anything new since the algorithmic part ends up receiving the same device data, just running the algorithms twice with the exact same data.

I removed 'dataframe' from the above code to make sure all the test files follow the logic

Reply:
If it is already being tested, that's very reasonable.